Rendering exception when trying to fill out the user task form template in native mode.

Hi,

The Qute engine can't fill out the user task form in native mode.

### Environment
**Versions**
Quarkus: 3.28.4
automatiko: 0.46.0

Command line to compile the container image
```sh
mvn package -Dmaven.test.skip=true -Dnative -Dquarkus.native.container-build=true -Dquarkus.container-image.build=true
```

### Exception
```log
Caused by: io.quarkus.qute.TemplateException: Rendering error in template [default-task-template.html:176]: Property "phase" not found on the base object "io.automatiko.engine.workflow.BaseWorkItem" in expression {task.phase}
        at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:173)
        at io.quarkus.qute.EvaluatorImpl.propertyNotFound(EvaluatorImpl.java:234)
        at io.quarkus.qute.EvaluatorImpl.resolve(EvaluatorImpl.java:204)
        at io.quarkus.qute.EvaluatorImpl.resolveReference(EvaluatorImpl.java:131)
        at io.quarkus.qute.EvaluatorImpl.lambda$resolveReference$2(EvaluatorImpl.java:135)
        at io.quarkus.qute.CompletedStage.thenCompose(CompletedStage.java:249)
        at io.quarkus.qute.EvaluatorImpl.resolveReference(EvaluatorImpl.java:135)
        at io.quarkus.qute.EvaluatorImpl.evaluate(EvaluatorImpl.java:85)
        at io.quarkus.qute.ResolutionContextImpl.evaluate(ResolutionContextImpl.java:28)
        at io.quarkus.qute.IfSectionHelper$OperandCondition.evaluate(IfSectionHelper.java:253)
        at io.quarkus.qute.IfSectionHelper$DoubletonCondition.evaluate(IfSectionHelper.java:292)
        ... 37 more
```

### Cause
According to the official Qute [guide](https://quarkus.io/guides/qute-reference#native_executables), to fill out the template, I should add the RegisterForReflection annotation when compiling the native image.

### Suggested changes
The PR #816

--- a/quarkus-extension/common/src/main/java/io/automatiko/engine/quarkus/ui/FormRelfectionConfig.java
+++ b/quarkus-extension/common/src/main/java/io/automatiko/engine/quarkus/ui/FormRelfectionConfig.java
@@ -1,0 +1,10 @@
+package io.automatiko.engine.quarkus.ui;
+
+import io.automatiko.engine.api.workflow.WorkItem;
+import io.automatiko.engine.workflow.BaseWorkItem;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection(targets = { WorkItem.class, BaseWorkItem.class })
+public class FormRelfectionConfig {
+
+}

